MiniApp/Classes/core/Models/MiniAppManifest.swift

```diff
@@ -0,0 +1,57 @@
+/// Mini App Meta data information
+internal struct MetaDataResponse: Decodable {
+    var bundleManifest: MetaDataCustomPermissionModel
+
+    private enum CodingKeys: String, CodingKey {
+        case bundleManifest
+    }
+}
+
+/// Mini App Meta data information
+internal struct MetaDataCustomPermissionModel: Decodable {
+    var reqPermissions: [MACustomPermissionsResponse]?
+    var optPermissions: [MACustomPermissionsResponse]?
+
+    var exampleHostAppMetaData: [String: String]?
```

Comment:
Sam here, this should be `customMetaData`.

Reply:
Done